lib/nsq-cluster/nsqd.rb

```diff
@@ -10,17 +10,20 @@ class Nsqd < ProcessWrapper
 
 
   def initialize(opts = {})
-    @host = opts[:host] || '127.0.0.1'
-    @tcp_port = opts[:tcp_port] || 4150
-    @http_port = opts[:http_port] || 4151
-    @lookupd = opts[:nsqlookupd] || []
-    @msg_timeout = opts[:msg_timeout] || '60s'
```

Comment:
@DanielleSucher I believe the reason we got rid of `msg_timeout` here is because you can specify it on a per connection basis: https://github.com/wistia/nsq-ruby/blob/master/lib/nsq/connection.rb#L31